card/mod/machines/lib/javascript/wagn.js.coffee

```diff
@@ -385,3 +385,7 @@ snakeCase = (str)->
               match[1].toLowerCase()
 
 warn = (stuff) -> console.log stuff if console?
+
+wagn.slotReady (slot) ->
+  slot.find('._disappear').delay(5000).animate(
```

Comment:
so `_disappear` is a class?  I may have started the practice of using CSS classes as javascript signifiers, but is that considered good practice?